setup.sh

```diff
@@ -96,14 +138,36 @@ apply () {
         else
                 $TERRAFORMPATH apply --var-file=$TERRAFORMTFVARS --auto-approve
         fi
+
+        # For development stacks, for the `infra-networking-route53` project, ensure that
+        # the shared_dev_route53 resource has been imported into terraform before applying
```

Comment:
This comment I think is slightly wrong. Maybe you could help @kentsanggds. 

At the moment if you are applying the `infra-networking-route53` project for the first time you will
- apply
- import_shared_dev_route53
- apply again

why do we want to apply twice as opposed to just doing:
- import_shared_dev_route53
- apply again

Reply:
A state file needs to be present otherwise it won't do the import, hence create the state file, then import and apply

Reply:
Thanks for the clarity